### PR TITLE
Add skipping of commented or empty lines

### LINKED ISSUE #36: Filter out commented lines
I think filtering should be realized before all checks, here: https://github.com/mgrachev/dotenv-linter/blob/9e28fcb41608d03181f01a444802b193725d3867/src/lib.rs#L36-L43

For now `incorrect_delimiter` check fails on line:
```env
#FOO=BAR
```
with message `.env:1 The #FOO key has incorrect delimiter`

Inline comments may affect some checks (but I don't investigate it):
```env
FOO=BAR # some comment ===
```

### PR DESCRIPTION
**Pull request by mstruebing**
For this I ignore inline comments, this can be implemented at a later stage.

closes #36

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### 🚀 Added
+- Add skipping of commented or empty lines [#36](https://github.com/mgrachev/dotenv-linter/pull/36) ([@mstruebing](https://github.com/mstruebing))
```
Comment (mgrachev) [inline CHANGELOG.md]:
```suggestion
- Add skipping of commented or empty lines [#37](https://github.com/mgrachev/dotenv-linter/pull/37) ([@mstruebing](https://github.com/mstruebing))
```

You should use the number of your PR.

**Comment by mgrachev** [comment]
@mstruebing Thank you for your contributing!

I've left a comment. Could you please fix it?

**Comment by artem-russkikh** [comment]
> For this I ignore inline comments, this can be implemented at a later stage.
> 
> closes #36

So this PR should not close the issue because it's partially completed

**Comment by mgrachev** [comment]
@artem-russkikh I suggest opening another issue for unrealized things. What do you think about it?

**Comment by artem-russkikh** [comment]
@mgrachev yes, that's better

I'll update and open new issue

**Comment by mgrachev** [comment]
@mstruebing Thank you very much! ❤️

**Pull request merged**